Assets/Scripts/DrawDetector.cs

```diff
@@ -1 +1,20 @@
+using UnityEngine;
 
+public class DrawDetector : MonoBehaviour
+{
+    [SerializeField] private GameCalculator _gameCalculator;
+
+    [SerializeField] private TextOutput _textOutput;
+
+    [SerializeField] private MoveCounter _moveCounter;
+
+    public const int NumberOfMove = 9;
+
+    public void СheckForADraw()
+    {
+        if (_moveCounter.Count != NumberOfMove || _gameCalculator.GameStates != GameState.Game) return;
+        _gameCalculator.GameStates = GameState.Draw;
+
+        _textOutput.ShowTextOfWinner(GameState.Draw);
```

Comment:
this class should check if game is a draw. And if it is it should return true and false otherwise. Right now it is not only checking for that, but also setting calls _textOutputMethod